lib/stack_master/cli.rb

```diff
@@ -215,6 +215,17 @@ def execute!
         end
       end
 
+      command :drift do |c|
+        c.syntax = 'stack_master drift [region_or_alias] [stack_name]'
+        c.summary = 'Detects and displays stack drift'
```

Comment:
Is it assumed here that "displays stack drift" means per Cloudformation standards and not what stack master has?

Reply:
Yup I have similar question, I have not looked at the code but I assume this is a drift between the current aws stack definition vs the actual stack resources and config. (i.e not the stack master code)

P.S - big +1 to including this stuff 🏆

Reply:
I assumed as much too but do wonder whether other users of this are on the same page.

Reply:
How about tweaking the summary to be more explicit?
```suggestion
        c.summary = 'Detects and displays stack drift using the CloudFormation Drift API'
```

Reply:
Yep I think we need documentation to clarify to users clearly what the difference between `diff` and `drift` is, as it's easily confused.